templates/shared_services/check_sp.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Can you add generic bash options to better handle issues?
```bash
set -o errexit
set -o pipefail
set -o nounset
```

Reply:
i've gone with `set -e` as that's what the majority of the other scripts use